sydent/db/invite_tokens.py

```diff
@@ -20,16 +20,46 @@ class JoinTokenStore(object):
     def __init__(self, sydent):
         self.sydent = sydent
 
-    def storeToken(self, medium, address, roomId, sender, token):
+    def storeToken(self, medium, address, roomId, sender, token, originServer=None, originId=None, commit=True):
+        """Stores an invite token.
+        
+        :param medium: The medium of the token.
+        :type medium: str
+        :param address: The address of the token.
+        :type address: str
+        :param roomId: The room ID this token is tied to.
+        :type roomId: str
+        :param sender: The sender of the invite.
+        :type sender: str
+        :param token: The token itself.
+        :type token: str
+        :param originServer: The server this invite originated from (if coming from replication).
+        :type originServer: str
```

Comment:
`:type arg1: int, float,...` apparently https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

But yes, good point

Reply:
I'm not sure that is meant to be taken literally. I'm also not sure that page is in any way authoritative, though you may know otherwise.

However, I now realise that our convention of using | appears to be something we have made up, so <shrugface>